newIDE/app/src/UI/Search/BoxSearchResultsNoScroll.js

```diff
@@ -0,0 +1,155 @@
+// @flow
+import { Trans } from '@lingui/macro';
+import * as React from 'react';
+import PlaceholderLoader from '../PlaceholderLoader';
+import PlaceholderError from '../PlaceholderError';
+import ErrorBoundary from '../ErrorBoundary';
+import { AutoSizer, Grid } from 'react-virtualized';
+import EmptyMessage from '../EmptyMessage';
+import {
+  GridListTile,
+  GridList,
+  makeStyles,
+  createStyles,
+} from '@material-ui/core';
+import Text from '../../UI/Text';
+import { Column, Line, Spacer } from '../../UI/Grid';
+import { shouldValidate } from '../../UI/KeyboardShortcuts/InteractionKeys';
+
+const styles = {
+  container: { flex: 1 },
+  grid: { overflowX: 'hidden' },
+};
+
+type Props<SearchItem> = {|
+  searchItems: ?Array<SearchItem>,
+  renderSearchItem: (item: SearchItem, size: number) => React.Node,
+  error: ?Error,
+  onRetry: () => void,
+  baseSize: number,
+  noResultPlaceholder?: React.Node,
+  onOpenDetails: (assetShortHeader: AssetShortHeader) => void,
+|};
+
+// The dimensions are retrieved using this solution:
+// https://stackoverflow.com/a/70608516
+const ResponsiveGrid = React.forwardRef(
+  (
+    {
+      searchItems,
+      renderSearchItem,
+      error,
+      onRetry,
+      baseSize,
+      noResultPlaceholder,
+      onOpenDetails,
+    }: Props<SearchItem>,
+    ref
+  ) => {
+    const [width, setWidth] = React.useState(100);
+    const [height, setHeight] = React.useState(100);
+
+    React.useEffect(
+      () => {
+        const resizeObserver = new ResizeObserver(event => {
+          // Depending on the layout, you may need to swap inlineSize with blockSize
+          // https://developer.mozilla.org/en-US/docs/Web/API/ResizeObserverEntry/contentBoxSize
+          setWidth(event[0].contentBoxSize[0].inlineSize);
+          setHeight(event[0].contentBoxSize[0].blockSize);
+        });
+
+        if (ref && ref.current) {
+          resizeObserver.observe(ref.current);
+        }
+      },
+      [ref]
+    );
```

Comment:
The comment from StackOverflow suggests react-use. It would make this simpler but it's a new dependency to add.

Reply:
It's hard to recommend any solution without actually trying to implement it. :)
I'll try to take a look tomorrow, to see if I find a more elegant solution.

It's too bad that we can't just re-use the BoxSearchResults by adding an additional prop to prevent scrolling